src/OpenConext/ProfileBundle/Form/Type/SwitchLocaleType.php

```diff
@@ -104,6 +106,6 @@ private function formatLocaleChoices(LocaleSet $availableLocales)
             $localeChoices[$locale->getLocale()] = 'profile.locale.' . $locale->getLocale();
         }
 
-        return $localeChoices;
+        return array_flip($localeChoices);
```

Comment:
What is the reason for flipping this array now? 

What about switching the keys and values when assigning it in the for each loop above?

Reply:
Yes, one operation less to deal with. 

While adding the flip function, my consideration was that it looked slightly better, not having the string concatenation in the array key.